test_conformance/basic/test_imagedim.cpp

```diff
@@ -38,24 +38,25 @@ static const char *image_dim_kernel_code =
 "}\n";
 
 
-static unsigned char *
-generate_8888_image(int w, int h, MTdata d)
+static unsigned char *generate_8888_image(size_t w, size_t h, MTdata d)
 {
-    unsigned char   *ptr = (unsigned char*)malloc(w * h * 4);
-    int             i;
+    unsigned char *ptr = new unsigned char[sizeof(unsigned char) * 4 * w * h];
```

Comment:
Minor: don't need `sizeof(unsigned char)` here anymore since the allocation is done with `new[]`.

Reply:
Addressed.

Reply:
Thanks!